sql/synth_recipes.sql

```diff
@@ -3113,7 +3113,7 @@ INSERT INTO `synth_recipes` VALUES (45009,0,0,0,0,0,50,103,0,0,0,4099,4241,869,2
 INSERT INTO `synth_recipes` VALUES (45010,0,0,0,0,0,0,104,0,0,0,4103,4245,635,3547,4509,0,0,0,0,0,3548,3548,3548,3548,1,1,1,1,'Sealord Leather');
 INSERT INTO `synth_recipes` VALUES (45011,0,0,0,0,0,0,104,0,0,0,4103,4245,695,3547,4509,0,0,0,0,0,3548,3548,3548,3548,1,1,1,1,'Sealord Leather');
 INSERT INTO `synth_recipes` VALUES (45012,0,0,0,0,0,0,104,0,0,0,4099,4241,1629,3552,3552,0,0,0,0,0,10546,10547,10547,10547,1,1,1,1,'Urja Gloves');
--- INSERT INTO `synth_recipes` VALUES (45013,0,0,0,255,255,0,104,0,0,0,4099,4241,665,862,862,914,1409,2189,2275,3925,27920,27921,27921,27921,1,1,1,1,'Pak Corselet');
+INSERT INTO `synth_recipes` VALUES (45013,0,0,0,62,255,0,104,0,0,0,4099,4241,665,862,862,914,1409,2189,2275,3925,27920,27921,27921,27921,1,1,1,1,'Pak Corselet');  -- 2021.11.17  FFXIclopedia and BGWiki show 104, JPWiki shows 104-105 Leathercraft.  BGWiki shows 60-62 Smithing.  All show ??? Goldsmithing.
```

Comment:
Leave this one commented out if it still has a 255 lvl

Reply:
guessing typo and meant to edit that not the 0

Reply:
No. It was an error on my end. There were a lot of 2 and 3 craft synths there. I didn't mean for that one to be uncommented. I'll get it fixed tonight.

Reply:
Concerns corrected.